### PR TITLE
Fix incorrect toolbar position after rotation

### PR DESCRIPTION
**Pull request by erysaj**
User can accidentally get into situation where toolbar is not at the bottom (or not visible at all) and keyboard is not active.
To reproduce dismiss keyboard immediately before rotating device (try in different orientation as behaviour slightly differs across OS)

This PR must be merged after #880 to properly work on iOS7

### CODE REVIEW

**Comment by jessesquires** [comment]
thanks @erysaj ! I will look soon!

File: JSQMessagesViewController/Controllers/JSQMessagesViewController.m [hunk 1]
```diff
@@ -301,6 +304,18 @@ - (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceO
     }
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
```
Comment (jessesquires) [inline JSQMessagesViewController/Controllers/JSQMessagesViewController.m]: this API is iOS 8.0-only

**Comment by erysaj** [comment]
Right, no problems unless trying to compile with SDK 7.x

**Comment by jessesquires** [comment]
Thanks @erysaj! 😄 

I think this was fixed by 752fd38aa8b899cbe2e21b6595de58920977bfc2

**Pull request closed without merging**